test/feature/StrongMode/ArityChecking.js

```diff
@@ -0,0 +1,94 @@
+// Options: --strong-mode --strong-mode-runtime
+
```

Comment:
It would be good to have some tests on other ways of invoking functions (Function.prototype.{call/apply/bind}, tagged template strings) and on function objects created in different ways (arrow functions, Function constructor).